web3pi_proxy/core/rpc/node/endpoint_pool/pool_manager.py

```diff
@@ -92,14 +90,114 @@ def check_connections(self, pools: List[EndpointConnectionPool]):
                     daemon=True,
                 ).start()
 
-            if self.__is_outdated(pool):
-                self.__logger.warning(f"Endpoint `{pool.endpoint.name}`: falling behind")
-
             self.__logger.debug(f"Endpoint `{pool.endpoint.name}`: okay.")
 
 
+class HttpResponseParserListenerSyncController:
```

Comment:
the name is "HttpResponseParserListenerSyncController" but then it's used as a "response listener" -> shouldn't the name reflect just that? 

```suggestion
class SyncControllerResponseListener:
```